src/common/common.cpp

```diff
@@ -715,3 +716,17 @@ bool check_user_is_admin()
     freeMemory(pSID, pGroupInfo);
     return false;
 }
+
+bool find_app_name_in_path(const std::wstring& where, const std::vector<std::wstring>& what)
+{
+    for (const auto& row : what)
+    {
+        const auto pos = where.rfind(row);
+        const auto last_slash = where.rfind('\\');
+        if (pos != std::wstring::npos && pos <= last_slash + 1 && pos + row.length() > last_slash)
```

Comment:
Nice one, I like it! Perhaps a comment clarifying what this does would be nice. Suggestion: Check that `row` occurs in `where`, and its last occurrence contains in itself the first character after the last backslash.

Reply:
Thank you, I'll add a comment